<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="lastStep">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextStep">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="camFrameView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="currentSideView">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>321</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="sixSideView">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>550</y>
     <width>701</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openCamera">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeCamera">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextFace">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>650</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Face</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
